bundles/org.eclipse.swt/Eclipse SWT PI/common/org/eclipse/swt/internal/Library.java

```diff
@@ -374,7 +374,7 @@ public static String getVersionString () {
 		}
 		version += MINOR_VERSION;
 		/* No "r" until first revision */
-		if (REVISION > 0) version += "r" + REVISION; //$NON-NLS-1$
+		//if (REVISION > 0) version += "r" + REVISION; //$NON-NLS-1$
```

Comment:
This change is not correct:
1. The revision must remain, it should only also be added for revision "0" (i.e., the `if` should be removed).
2. The comment should be adapted to conform to the code.

For the sake of comprehensibility, I would be in favor of putting this change at least into a separate commit with a proper commit description as it is independent from configuring build scripts for the release.